18-minimal/Dockerfile.fedora

```diff
@@ -0,0 +1,67 @@
+# using registry.fedoraproject.org because there is no f36 version of fedora-minimal on quay.io/fedora
+FROM registry.fedoraproject.org/fedora-minimal:38
```

Comment:
Does it exist also on quay.io? What about to use from this repository https://quay.io/repository/fedora/fedora-minimal

Reply:
good point, will do